src/native/corehost/hostmisc/pal.h

```diff
@@ -226,7 +229,9 @@ namespace pal
     inline void err_fputs(const char_t* message) { ::fputs(message, stderr); ::fputc(_X('\n'), stderr); }
     inline void out_vprintf(const char_t* format, va_list vl) { ::vfprintf(stdout, format, vl); ::fputc('\n', stdout); }
     inline int str_vprintf(char_t* str, size_t size, const char_t* format, va_list vl) { return ::vsnprintf(str, size, format, vl); }
-    inline const char_t* strerror(int errnum) { return ::strerror(errnum); }
+    inline int strlen_vprintf(const char_t* format, va_list vl) { return ::vsnprintf(nullptr, 0, format, vl); }
+
+    inline const string_t strerror(int errnum) { return ::strerror(errnum); }
```

Comment:
I think there are other places that need to be updated.

Reply:
Ugh. Guess who only built on windows and only searched for the fully qualified name.